packages/nvidia_nat_test/src/nat/test/tool_test_runner.py

```
@@ -289,6 +292,21 @@ def get_function_group_dependencies(self, fn_name: str) -> FunctionDependencies:
         """Mock implementation."""
         return FunctionDependencies()
 
+    async def get_middleware(self, middleware_name: str | MiddlewareRef) -> FunctionMiddleware:
```

Ruff (0.14.4)

Unused method argument: `middleware_name`

(ARG002)